### ISSUE TITLE
IPv6 is off, but AGH continues to listen to the port

### ISSUE THREAD

**Issue by ammnt** [AdguardTeam/AdGuardHome#2930]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 0.105.2
* **How did you install AdGuard Home:**
  * Github release
* **How did you setup DNS configuration:**
  * VPS
* **Operating system and version:**
  * Debian 10

### Actual Behavior
Adapter does not support IPv6. IPv6 is completely off, but AGH continues to listen to the port:
![image](https://user-images.githubusercontent.com/45385632/113889443-eae58e00-97cb-11eb-8ac8-b75d479e2807.png)

</details>

### Additional Information
What is it? Some socket? For what? Why?😅

**Comment by ainar-g** [comment]
Hi, just to be clear, by “IPv6 is off” do you mean that `aaaa_disabled` in the configuration file is set to `true` or just that the hardware or the OS don't support it?  Does it interfere with AGH's normal functioning, or is it just undesirable behaviour?

**Comment by ammnt** [comment]
> Hi, just to be clear, by “IPv6 is off” do you mean that `aaaa_disabled` in the configuration file is set to `true` or just that the hardware or the OS don't support it? Does it interfere with AGH's normal functioning, or is it just undesirable behaviour?

Yes, it's off at AGH through `aaaa_disabled` + my domain has no AAAA record + IPv6 is disabled via sysctl.conf:
```
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
net.ipv6.conf.eth0.disable_ipv6 = 1
net.ipv4.icmp_echo_ignore_all = 1
```

**Comment by ammnt** [comment]
@ainar-g , AGH is functioned normally, but i want to know what is it and i keep to close all unused ports and disable all unused daemons. Thank you.

**Comment by ainar-g** [comment]
I see, thanks.  `aaaa_disabled` merely disables responding to `AAAA` queries, regardless of which protocol they came over.  The part that controls which networks we're listening on is in one of our external libraries, so we'll probably won't be able to add this enhancement in this cycle, but we should probably do that some time after that.

**Comment by DandelionSprout** [comment]
Adding `::/0` to *Disallowed clients*, should in theory accomplish OP's goal, except that the admin interface would remain available on IPv6.

**Comment by ammnt** [comment]
@ainar-g, it works really weird:
![image](https://user-images.githubusercontent.com/45385632/115959136-4c6b6380-a513-11eb-8ea0-c3d85bcf800d.png)
Now he listens only on the IPv6 protocol. At the same time, the IPv6 is still turned off and all DNS queries are only IPv4. The loopback IPv6 is not existed too...🤔
